PointSDK-MinimalIntegrationExample-Swift/ViewController.swift

```diff
@@ -12,38 +12,114 @@ import BDPointSDK
 import UserNotifications
 
 class ViewController: UIViewController {
-    //Add API key for the App
-    var apiKey = ""
+    // Add valid Project Id for the App
```

Comment:
Shell we add a more descriptive comment? E.g.

> Use a project Id acquired from the Canvas UI.

If yes, Something similar should be for the `tempoDestinationId`